dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/ServiceManager.scala

```diff
@@ -124,8 +123,7 @@ class ServiceManager @Inject() (logger: MavenLoggerProxy, session: MavenSession,
               cassandraPort.fold(Map.empty[String, String]) { port =>
                 LagomConfig.cassandraPort(port)
               } ++
-              LagomConfig.cassandraKeySpace(cassandraKeyspace) ++
-              Map(LagomConfig.KafkaAddress -> kafkaAddress)
```

Comment:
I don't know why it's passing the Cassandra port - I don't think it should be, it just confuses things.